src/main/config/index.ts

```diff
@@ -37,3 +37,4 @@ export const NAME_REGISTERED_WITH_AML = `${BASE_COMMON_URL}/name-registered-with
 export const WHAT_IS_YOUR_ROLE = `${BASE_COMMON_URL}/what-is-your-role/what-is-your-role`;
 export const STOP_NOT_RELEVANT_OFFICER_KICK_OUT = `${BASE_COMMON_URL}/stop-not-relevant-officer/stop-not-relevant-officer`;
 export const SECTOR_YOU_WORK_IN = `${BASE_COMMON_URL}/sector-you-work-in/sector-you-work-in`;
+export const WHICH_SECTOR_YOTHER = `${BASE_COMMON_URL}/which-sector-other/which-sector-other`;
```

Comment:
Is there a typo here in the constant name?

Reply:
fixed it now